### DISCUSSION TITLE
Translation update details

### DISCUSSION THREAD

**Discussion by ruojianll** [facing-dev/vue-facing-decorator#53] [General]
In here, I will update the english doc update details. Please translate that to other languages. Please make a reply for every language translate and mark what time in detail list you have implemented and update the reply whenever you make a new translation update.

Detail list:
03/31/2023 base line
03/31/2023-2 Rename `class-component.use` to `class-component.setup` and add `class-component.component.setup` in english and zh-ch.
04/02/2023 Add `mixins` function to component inheritance. https://facing-dev.github.io/vue-facing-decorator/#/en/inheritance/component/component?id=extends-multiple-components , en & zh-cn

Portuguese @BrendonHenrique

**Comment by BrendonHenrique** [comment]
hey ya ! 
of course ! I will address this out ! and will be there asap !

**Comment by BrendonHenrique** [comment]
Hey @ruojianll 

Just updated pt-br docs with your last docs changes. 

04/29/2023 [Setup Doc](https://github.com/facing-dev/vue-facing-decorator/pull/61/commits/9b66004faa052eab7491b70b9a791b137460862f)
 
04/29/2023 [Mixins Functions](https://github.com/facing-dev/vue-facing-decorator/pull/61/commits/d43ab9be7d309f9be1706503c0994922d5eb7a7d)

04/29/2023 [CreateDecorator](https://github.com/facing-dev/vue-facing-decorator/pull/61/commits/0b42081bf8db4ce35c11f0a0b139579f6de1c92a)

**Reply by ruojianll** [reply]
Merged and published

**Comment by ruojianll** [comment]
@BrendonHenrique Hey buddy, glad to tell you that v3 has been released. Please sync the document:
07/30/2023
1. Renamed `next version` into `migrate from v2` in the top level and changed some text, please check it out.
2. New `stage3-decorators` section in the top level.
3. New `Use toNative` section at the end of `quick-start` section.
4. Changed almost all code examples, just reference it from english version, don't need to transform text in the examples. e.g. `../../en/path/to/example.ts`, please replace all code example references.
5. Remove all `Donate` sections.
6. Update `Class component > Component > Options > mixins`
7. Totally remove `Compatibility` section in the top level.
8. Update `Usage` and `Extends multiple components` in `Inheritance > Component`

**Comment by ruojianll** [comment]
@BrendonHenrique 
07/30/2023
Update `Custom Decorator` section in the top level.

**Comment by ruojianll** [comment]
@SaitoYuto
